### PR TITLE
full tupelo network in single docker image

### PR DESCRIPTION
**Pull request by brandonwestcott**
Kind of a big change in direction here, but I think this supports our needs and reduces having to manage a "codebase"

The idea here is to give the user a 'simple' local network and allow them to use the normal `docker-compose` to run their tests, rather than relying on custom code & config.

Functionally all this does is create a docker image that contains a [full tupelo network docker-compose](https://github.com/quorumcontrol/tupelo-local/blob/v2/docker/docker-compose.yml) with the required [toml configs](https://github.com/quorumcontrol/tupelo-local/tree/v2/docker/config) and a small [runner](https://github.com/quorumcontrol/tupelo-local/blob/v2/docker/run.sh). Then a user can run this image in their normal `docker-compose.yml`, then on start, this container will inspect its environment (by mounting `/var/run/docker.sock`), and will run a tupelo network thats accessible by the other containers in the parent stack.

See https://github.com/quorumcontrol/tupelo-local/blob/v2/README.md for a bit more description on usage.

Replaces https://github.com/quorumcontrol/tupelo-integration-runner/pull/19

### CODE REVIEW

File: docker/config/bootstrap.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Port = 34001
+BootstrapOnly = true
+NotaryGroupConfig = "./notarygroup.toml"
+
```
Comment (brandonwestcott) [inline docker/config/bootstrap.toml:4]: @tobowers regarding https://github.com/quorumcontrol/tupelo-integration-runner/pull/19#discussion_r349522322

Our other toml configs don't set it, but the ip is set manually on docker-compose so maybe it makes sense?

**Comment by brandonwestcott** [comment]
(renaming things real quick too)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+FROM debian:stretch-slim
+LABEL maintainer="dev@quroumcontrol.com"
```
Comment (cap10morgan) [inline Dockerfile]: s/quroum/quorum/ :)

**Review by cap10morgan — APPROVED** [review]
One little typo to fix, but looks great otherwise!

**Pull request merged**